frontend/src/app/drive/layout.tsx

```diff
@@ -0,0 +1,96 @@
+import { Switch } from "@headlessui/react";
+import {
+  GlobeIcon,
+  HomeIcon,
+  ShareIcon,
+  TrashIcon,
+  UserIcon,
+} from "lucide-react";
+import type { Metadata } from "next";
+import localFont from "next/font/local";
+import { useLocalStorage } from "usehooks-ts";
+import { ScopeSwitch } from "../../components/common/ScopeSwitch";
+import { SearchBar } from "../../components/SearchBar";
+import "../globals.css";
+
+const geistSans = localFont({
+  src: "../fonts/GeistVF.woff",
+  variable: "--font-geist-sans",
+  weight: "100 900",
+});
+const geistMono = localFont({
+  src: "../fonts/GeistMonoVF.woff",
+  variable: "--font-geist-mono",
+  weight: "100 900",
+});
+
+export const metadata: Metadata = {
+  title: "Auto-Drive",
+  description: "Autonomys Drive",
+};
+
+export default function RootLayout({
```

Comment:
Is the layout of this page completely different?
Otherwise, you could use`Layout` instead of `RootLayout` to use more of an island architecture and only add layout needed for this specific section.

Reply:
I saw lower that you removed most of the code from the RootLayout, then here you could change for `Layout` and simply get rid of the already defined html and body tag and it should be gucci